### ISSUE TITLE
Phase 2 (Stage 1): Execute Wayfarer-12B Foundation SFT Pipeline

### ISSUE THREAD

**Issue by nochadisfaction** [pixelatedempathy/pixelated#542]
# Objective

Execute the rigorous Foundation Training algorithm representing 40% of the unified curriculum via `stage1_foundation.json`. This stage builds the deep empathy mapping over Wayfarer-12B's base topology.

## Requirements & Scope

* **Dataset**: Streams directly from `s3://pixel-data/final_dataset/` via `S3DatasetLoader`.
* **Target Base Model**: `LatitudeGames/Wayfarer-12B`.
* **Checkpoint Location**: Writes to `./therapeutic_ai_final_stage1`.

## LoRA Architecture Specification

* **Rank (**`r`): 16
* **Alpha**: 32
* **Dropout**: 0.05
* **Target Modules**: `q_proj`, `v_proj`
* **Context Length**: 1024 (Limited in Stage 1 to optimize broader base context absorption)
* **Epochs**: 3

## Execution Protocol

```bash
uv run python ai/lightning/production/train_therapeutic_ai.py --stage 1 --config ai/lightning/production/stage_configs/stage1_foundation.json
```

## Acceptance Criteria

- [ ] PyTorch Lightning training loops complete without throwing OOM (Out-of-Memory) faults across the \~3.9M canonical JSONL records.
- [ ] Evaluation `val_loss` converges stably across the 3 epochs with no gradient spikes.
- [ ] A fully populated `.safetensors` PEFT adapter is correctly exported to `./therapeutic_ai_final_stage1`.
